support/troubleshooting/troubleshooting-operator-issues.adoc

```diff
@@ -0,0 +1,26 @@
+[id="troubleshooting-operator-issues"]
+= Troubleshooting Operator issues
+include::modules/common-attributes.adoc[]
+:context: troubleshooting-operator-issues
+
+toc::[]
+
+Operators are a method of packaging, deploying, and managing an {product-title} application. They act like an extension of the software vendor’s engineering team, watching over an {product-title} environment and using its current state to make decisions in real time. Operators are designed to handle upgrades seamlessly, react to failures automatically, and not take shortcuts, like skipping a software backup process to save time.
+
+{product-title} {product-version} includes a default set of Operators that are required for proper functioning of the cluster. These default Operators are managed by the Cluster Version Operator (CVO).
+
+As a cluster administrator, you can install application Operators from the OperatorHub using the {product-title} web console or the CLI. You can then subscribe the Operator to one or more namespaces to make it available for developers on your cluster. Application Operators are managed by the Operator Lifecycle Manager (OLM).
```

Comment:
Throughout this assembly and its modules, for consistency with the other OLM docs:

s/the Operator Lifecycle Manager/Operator Lifecycle Manager/

Reply:
Resolved throughout the PR.